classes/assets/CccReducer.php

```diff
@@ -45,7 +52,12 @@ public function __construct($cacheDir, ConfigurationInterface $configuration, Fi
         }
     }
 
-    public function reduceCss($cssFileList)
+    /**
+     * @param array $cssFileList
+     *
+     * @return array Same list, reduced
+     */
+    public function reduceCss(array $cssFileList)
```

Comment:
It's a BC :thinking: 
What if the parameter was `ArrayAccess` or something similar?

Reply:
Ay 😅 I didnt think about `ArrayAccess`

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/18185/commits/e495740557e289cac08d92f6319badd4233dbd63